src/lib/private_key.js

```diff
@@ -43,7 +43,7 @@ function validateDecryptedData(taskPayload, decryptedData, taskId) {
                                 errorPrefix, logMsg, incidentId));
   }
 
-  if (_.contains(reservedKeys, decryptedData.name)) {
+  if (_.includes(reservedKeys, decryptedData.name)) {
```

Comment:
If you want to reduce lodash dependencies, this can also be `reservedKeys.includes(decrypedData.name)`.